ts/src/kucoinfutures.ts

```diff
@@ -995,6 +996,65 @@ export default class kucoinfutures extends kucoin {
         return this.parsePositions (data, symbols);
     }
 
+    async fetchPositionsBySymbol (symbol, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        if (!market['linear'] || !market['swap']) {
+            throw new NotSupported (this.id + ' fetchPositionsBySymbol() is not yet supported for ' + symbol + ' market. Coming soon...');
+        }
+        const request = {
+            'symbol': market['id'],
+        };
+        const response = await (this as any).futuresPrivateGetPosition (this.extend (request, params));
```

Comment:
@ttodua why `this as any` here? it that is needed now something ain't right

Reply:
ah, as these were too much old PR (which I had only dissected and copy&pasted into new PRs) I missed to correct those olds things. i'll push updates.